bin/build_package.js

```diff
@@ -70,7 +70,12 @@ async function main() {
 
         console.log('Minifying CSS...');
         const css = await fs.promises.readFile(inputStyleFile, 'utf-8');
-        const minified = new CleanCSS().minify(css).styles;
+        const { code: minified } = LightningCSS.transform({
+            filename: path.basename(inputStyleFile, '.css'),
+            code: Buffer.from(css),
+            minify: true,
+            sourceMap: false, // TODO: Maybe we can add source maps later? :)
```

Comment:
Source Maps have more interest when there is multiple source files, variables, etc.. not the case here.

But if we have a component offering CSS, why not